gammapy/estimators/points/core.py

```diff
@@ -370,6 +370,8 @@ def to_table(self, sed_type=None, format="gadf-sed", formatted=False):
                 table["stat_scan"] = self.stat_scan.data[idx]
 
             table["is_ul"] = self.is_ul.data[idx]
+            if not self.has_ul:
+                table.remove_columns("is_ul")
```

Comment:
I think this would be even simpler:

```suggestion
            if self.has_ul:
                table["is_ul"] = self.is_ul.data[idx]
```

Reply:
This was also my initial idea, however, this section of code already creates the column `is_ul` and fills that column with data.

https://github.com/gammapy/gammapy/blob/0655a46e352c8d4ccf38168e89d3b7bb06b038c4/gammapy/estimators/points/core.py#L361-L364

Open to further suggestion

Reply:
You are right, just leave the suggestion open....